docs-api/frontity-packages/features-packages/wp-comments.md

```diff
@@ -0,0 +1,226 @@
+
+###### The _fields_ of a comment
+
+| Name          | Type   | Required | Description                                                    |
+| ------------- | ------ | -------- | -------------------------------------------------------------- |
+| `content`     | string | yes      | Content of the comment.                                        |
+| `authorName`  | string | no       | Author's name.                                                 |
+| `author`      | number | no       | The ID of the author.                                          |
+| `authorEmail` | string | no       | Author's email.                                                |
+| `authorURL`   | string | no       | URL of the author's site.                                      |
+| `parent`      | number | no       | ID of the comment to which this one responds. Default Value: 0 |
+
+##### `state.comments.forms[].errors`
+
+The validation errors returned from WordPress REST API are stored in the state in `state.comments.forms[].errors`. Each field sent as part of the comments object will have its related property under the `state.comments.forms[].errors` object if there's an error related to that field.
+
+Full list of fields that may be under this object can be seen at [The _fields_ of a comment](#the-fields-of-a-commentthe-fields-of-a-comment)
+
+#### `state.source.comment`
+
+This is the portion of the state where the comments are stored after being fetched from the REST API or POSTed through the `comments.submit()` action
+
+Thanks to the handler `@comments/:id` you can get the [ID's of the comments](https://github.com/frontity/frontity/blob/2eb98ae4e6fee1f93ac5af5c834a3add644ba7b0/packages/wp-comments/types.ts#L158) in a specific post.
+
+With this list of ID's you can get the details for each one at `state.source.comment[id]`
+
+_Example_
+
+```js
+const data = state.source.get(`@comments/${postId}`);
+
+data.items
+  .map(({ id }) => {
+    // For each ID we can get the details of each comment at state.source.comment[id]
+    const authorName = state.source.comment[id].author_name || "Anonymous";
+    const content = state.source.comment[id].content.rendered;
+    const date = state.source.comment[id].date;
+    return { id, authorName, content, date };
+  })
+  .forEach(console.log);
+```
+
+Check a fully working example of [this](https://github.com/frontity-demos/frontity-examples/blob/master/wp-comments/packages/mars-theme/src/components/comments/comments-list.js) in [this `wp-comments` demo](https://github.com/frontity-demos/frontity-examples/tree/master/wp-comments).
+
+### Actions
+
+#### `actions.comments.updateFields()`
+
+Update the fields of the form specified by `postId`. This action simply updates what is stored in `state.comments.form[postId].fields` with the given values.
```

Comment:
It's not clear what the purpose of this is. Shouldn't the fields contain the values of the comment form fields? Why would you need to change or override these?

Reply:
As #21 (comment)
Let me try to improve the explanation